packages/medusa/src/api/routes/admin/uploads/index.ts

```diff
@@ -52,14 +52,21 @@ export default (app) => {
  *       type: object
  *       required:
  *         - url
+ *         - key
  *       properties:
  *         url:
  *           description: The URL of the uploaded file.
  *           type: string
  *           format: uri
+ *         key:
+ *           description: The key of the file that is identifiable by the file service. It can be used later to retrieve or manipulate the file.
+ *           type: string
  */
 export type AdminUploadsRes = {
-  uploads: { url: string }[]
+  uploads: {
+    url: string
+    key: string
+  }[]
```

Comment:
suggestion: Use `FileServiceUploadResult` from `@medusajs/types`
```suggestion
    uploads: FileServiceUploadResult[]
```